### PR TITLE
feat(IAM Policy Management): re-gen service after fix the template state parameter

### PR DESCRIPTION
**Pull request by Rajesh-Pirati**
## PR summary
issue: https://github.ibm.com/IAM/AM-issues/issues/1949

## PR Checklist
Please make sure that your PR fulfills the following requirements:  
- [x] The commit message follows the [Angular Commit Message Guidelines](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-guidelines).
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## Does this PR introduce a breaking change?    
- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by Rajesh-Pirati** [comment]
integration tests:
image

Examples:
image

**Review by swcolley — COMMENTED** [review]
Looks good, curious about integration tests though.

**Comment by Rajesh-Pirati** [comment]
Integration test results after updating the testcases to validate the state parameter
image

**Review by swcolley — APPROVED** [review]
Looks good.

File: modules/iam-policy-management/src/main/java/com/ibm/cloud/platform_services/iam_policy_management/v1/model/CreateV2PolicyOptions.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
- * (C) Copyright IBM Corp. 2023.
+ * (C) Copyright IBM Corp. 2024.
```
Comment (padamstx) [inline modules/iam-policy-management/src/main/java/com/ibm/cloud/platform_services/iam_policy_management/v1/model/CreateV2PolicyOptions.java:2]: Although it doesn't cause any harm to include these changes in the PR, there's really no *need* to commit changes like this where a new class differs with the old only by the copyright year.

**Review by padamstx — APPROVED** [review]
LGTM

**Pull request merged**